.ci/pytorch/test.sh

```diff
@@ -374,6 +379,10 @@ test_inductor_huggingface() {
   test_inductor_benchmark huggingface ""
 }
 
+test_inductor_huggingface_cpu_accuracy() {
```

Comment:
Can we combine this with `test_inductor_huggingface` and add a device parameter? The same for the other two benchmarks.